angular-ngrx-scss/src/app/shared/components/issue-pull-item/issue-pull-item.component.html

```diff
@@ -0,0 +1,48 @@
+<div *ngFor="let item of itemsList">
+  <div class="item">
+    <div class="item-container">
+      <div class="info">
+        <ng-container *ngIf="item.state === 'open'; else closed">
+          <!-- no-entry -->
+          <span class="icon {{ item.state }}" appOcticon="no-entry"></span>
+        </ng-container>
+        <ng-template #closed>
+          <!-- issue-closed -->
+          <span class="icon {{ item.state }}" appOcticon="issue-closed"></span>
+        </ng-template>
+        <div>
+          <div class="content">
+            <span class="title">{{ item.title }}</span>
+            <span class="inline" *ngFor="let label of item.labels">
+              <span
+                [style.background-color]="colorMap(label.color)"
```

Comment:
A super nice-to-have, but it would be cool to have some method that calculates the bg brightness and changes the text color to white if the color is "too dark". [Inspiration here](https://gist.github.com/w3core/e3d9b5b6d69a3ba8671cc84714cca8a4) 

The black text on purple background doesn't work so great visually imo: <img width="80" alt="image" src="https://user-images.githubusercontent.com/11267785/182200520-5738928f-f5d3-40a8-ae5f-6bc58086f4cf.png">

Reply:
Made a ticket for this as well, agreed that this would be nice to implement: https://github.com/thisdot/starter.dev-github-showcases/issues/512